<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1020</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PaPen</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>600</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>970</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(64, 64, 64);
color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>三维重建</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>890</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(64, 64, 64);
color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>采集卡</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>600</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>655</x>
     <y>620</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>975</x>
     <y>620</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>655</x>
     <y>820</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>620</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QVTKWidget" name="qvtkWidget" native="true">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>10</y>
     <width>600</width>
     <height>1000</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>930</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(64, 64, 64);
color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>触发器</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>850</y>
     <width>170</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(64, 64, 64);
color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>摄像头</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>690</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>截取数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>690</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>截取点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>660</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>采样率</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>765</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(64, 64, 64);
color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>选择存储路径</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>690</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(32, 32, 32);
color: rgb(128, 128, 128);</string>
   </property>
   <property name="maximum">
    <number>2048</number>
   </property>
   <property name="value">
    <number>400</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>660</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(32, 32, 32);
color: rgb(128, 128, 128);</string>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>250</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>690</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(32, 32, 32);
color: rgb(128, 128, 128);</string>
   </property>
   <property name="maximum">
    <number>4096</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>805</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(128, 128, 128);</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>810</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>存</string>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_2">
   <property name="geometry">
    <rect>
     <x>1895</x>
     <y>10</y>
     <width>20</width>
     <height>580</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>20</width>
     <height>580</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>975</x>
     <y>820</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>810</y>
     <width>630</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(96, 96, 96);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>0</y>
     <width>1</width>
     <height>1020</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(96, 96, 96);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>0</y>
     <width>1</width>
     <height>1020</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(96, 96, 96);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>615</y>
     <width>630</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(96, 96, 96);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>615</y>
     <width>645</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(96, 96, 96);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>630</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(32, 32, 32);
color: rgb(128, 128, 128);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>630</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>串口号</string>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>615</y>
     <width>1</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(96, 96, 96);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>965</x>
     <y>615</y>
     <width>1</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(96, 96, 96);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>660</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>量程</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>720</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>深切终</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>720</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>深切始</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>720</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(32, 32, 32);
color: rgb(128, 128, 128);</string>
   </property>
   <property name="maximum">
    <number>2048</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>720</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(32, 32, 32);
color: rgb(128, 128, 128);</string>
   </property>
   <property name="maximum">
    <number>2048</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>399</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>660</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(32, 32, 32);
color: rgb(128, 128, 128);</string>
   </property>
   <item>
    <property name="text">
     <string>200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>500</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2000</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>197</x>
     <y>854</y>
     <width>30</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
